### ISSUE TITLE
Cannot install `bezier` with pip install

### ISSUE THREAD

**Issue by frenchqwerty** [dhermes/bezier#243]
Hello,

Env : 
Python 3.7
Debian 10

I am trying to instal bezier with cmd `sudo pip install bezier` and I have an error : 
```
Collecting bezier
  Using cached https://files.pythonhosted.org/packages/b9/0c/b1982f93c36fc06850a3880bb693bf24a047946633ca7403664836b9da9e/bezier-2020.5.19.tar.gz
    Complete output from command python setup.py egg_info:
    The BEZIER_INSTALL_PREFIX environment variable must be set.
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-1zufqw6_/bezier/

```

**Comment by sanyamN** [comment]
try not use sudo

**Comment by dhermes** [comment]
I suppose this is caused by the fact that I don't have Python 3.9 wheels, have you recently upgraded to Python 3.9? (If yes, bravo on living on the cutting edge 👏 .)

----

For "generic" installing from source instructions:

- https://bezier.readthedocs.io/en/2020.5.19/abi/installation.html#installation for instructions on how to install `libbezier` (the shared library) at some path e.g. `"${HOME}/libbezier-release/usr"`
- https://bezier.readthedocs.io/en/2020.5.19/python/binary-extension.html#source for how to install the Python package once `libbezier` has been installed

----

Alternatively, you disable the Python binary extension that utilizes `libbezier` by using

```
$ BEZIER_NO_EXTENSION=true \
>   python  -m pip install --upgrade bezier --no-binary=bezier
```

(https://bezier.readthedocs.io/en/2020.5.19/index.html#installing)

**Comment by dhermes** [comment]
I noticed you said "Python 3.7" so I'm now curious why there is no wheel. But maybe `sudo` has a different default than your user? Can you compare output of:

```
$ python -V
$ sudo python -V
```

you may actually have Python 3.5 (vs. Python 3.9) as the root user's default `python`.

**Comment by frenchqwerty** [comment]
Command `python -V` and `sudo python -V` give me Python 3.7.3
I have not intalled python 3.9, I am currently install bezier with pip install and I am not understand why I have that error... I am a newbie with python. I think a dependance is not installed.

Now I am trying to install libbezier according your response @dhermes

**Comment by frenchqwerty** [comment]
I installed the package from source and bezier is now installed. How can I install bezier in venv ?

**Comment by AndyEngland521** [comment]
Hello, I am also having the issue where I cannot pip install bezier on Python 3.9. Trying it without the extension works but then my script won't run as it can't find the DLL

**Comment by dhermes** [comment]
@AndyEngland521 thanks for the update. I need to cut a release (my last release was before 3.9 was released).

**Comment by AndyEngland521** [comment]
@dhermes all good :) figured I'd confirm since @frenchqwerty was on 3.7

**Comment by dhermes** [comment]
Fixed by `2021.2.12` release. (See #246, e.g.)
